libass/ass_render.h

```diff
@@ -208,6 +208,7 @@ typedef struct {
     ASS_Renderer *renderer;
     TextInfo *text_info;
     ASS_Shaper *shaper;
+    RasterizerData *rasterizer;
```

Comment:
Why an additional level of indirection? Just put `RasterizerData` here directly, also less allocations in the last commit.